include/pelib/PeFile.h

```diff
@@ -696,6 +700,35 @@ namespace PeLib
 		return ERROR_DIRECTORY_DOES_NOT_EXIST;
 	}
 
+	template<int bits>
+	LoaderError PeFileT<bits>::checkEntryPointErrors() const
+	{
+		unsigned int uiEntryPointRva = peHeader().getAddressOfEntryPoint();
+		std::uint64_t uiOffset = peHeader().rvaToOffset(uiEntryPointRva);
+		std::uint64_t entryPointCode[2];
```

Comment:
Consider adding a comment why we need to check 16 bytes. I assume it is because the entrypoint instruction can be up to 15 bytes long, and everything longer will result into an error when trying to run such a program.